src/qt/radiobut.cpp

```diff
@@ -59,3 +70,40 @@ QWidget *wxRadioButton::GetHandle() const
 {
     return m_qtRadioButton;
 }
+
+void wxRadioButton::CreateAndJoinNewGroup()
+{
+    QButtonGroup* qtButtonGroup = new QButtonGroup( GetHandle() );
+    qtButtonGroup->addButton( m_qtRadioButton );
+}
+
+void wxRadioButton::SearchForPreviousGroupToJoin( wxWindow *parent )
+{
+    wxWindowList::compatibility_iterator node = parent->GetChildren().GetLast();
```

Comment:
Why do we start from the last window? We should start from this one (or, rather, the one immediately before it). See wxMSW code which does

        wxWindowList::compatibility_iterator nodeThis = siblings.Find(this);